website_snippet_marginless_gallery/views/snippets.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2015-2017 Tecnativa - Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2019 Tecnativa - Cristina Martin R.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+<odoo>
+
+    <template id="s_marginless_gallery" name="Marginless Gallery">
+        <section class="marginless-gallery pt32 pb32 text-center">
+            <div class="container">
+                <div class="row">
+                    <div class="col-lg-12 text-center">
+                        <h2 class="o_default_snippet_text pb16"><b>Our Gallery</b></h2>
+                        <p class="lead o_default_snippet_text pb32">
+                            All photos must have the same height or the
+                            gallery layout will be broken
+                        </p>
+                    </div>
+                    <section class="col-lg-3 col-12 pt32 pb32 oe_img_bg" style="background-image: url('/website/static/src/img/library/bridge.jpg');">
```

Comment:
I tested that in runbot and it doesn't work. It probably needs more work to be done. It could be a good addition, but I don't think it's a requirement for migrating the addon.